src/app/component/auth/new-sign-up/new-sign-up.component.html

```diff
@@ -19,86 +19,185 @@
     <div class="main-content-container">
       <div class="title">
         <h1 class=""title-text>
-          Вітаємо!
+          {{ 'user.auth.sign-up.title-up' | translate }}
```

Comment:
Please fix it
![image](https://user-images.githubusercontent.com/54113846/79224771-63896700-7e64-11ea-816f-99ab1dbe6786.png)

Reply:
Fixed